bundles/org.openhab.binding.mielecloud/README.md

```diff
@@ -0,0 +1,648 @@
+| ----------- | --------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------ |
+| locale      | mandatory | The locale to use for full text channels of things from this account. Possible values are `en`, `de`, `da`, `es`, `fr`, `it`, `nl`, `nb`. Default is `en`. | Prior to the 7th January 2021 `da`, `es`, `fr`, `it`, `nl` and `nb` will default to English. |
+
+Example things-file:
+```
+Bridge mielecloud:account:home [ locale="en" ]
+```
+
+### Appliance Configuration
+
+The binding configuration UI will show a things-file template containing things for all supported appliances from the paired account.
+This can be used as a starting point for a custom things-file.
+All Miele cloud appliance things have no configuration parameters.
+
+Example things-file:
+```
+Bridge mielecloud:account:home [ locale="en" ] {
+    Thing coffee_system 000703261234 "Coffee machine CVA7440" [ ]
+    Thing hob 000160102345 "Cooktop KM7677" [ ]
+    Thing washing_machine 000148503456 "Washing Machine WWV980" [ ]
+    Thing oven 000137434567 "Oven" [ ]
+    Thing fridge 007109235678 "Refrigerator K34483" [ ]
+    Thing dishwasher 000159456789 "Dishwasher G5000" [ ]
+    Thing fridge_freezer 711397890 "Fridge-freezer" [ ]
+    Thing dryer 000091098901 "Dryer" [ ]
+}
+```
+
+## Channels
+
+The following table lists all available channels.
+See the following chapters for detailed information about which appliance supports which channels.
+Depending on the exact appliance configuration not all channels might be supported, e.g. a hob with four plates will only fill the channels for plates 1-4.
+
+| Channel Type ID               | Item Type            | Description                                                                                                                               | Read only |
+| ----------------------------- | -------------------- | ----------------------------------------------------------------------------------------------------------------------------------------- | --------- |
+| remote_control_can_be_started | Switch | Indicates if this device can be started remotely. | Yes |
+| remote_control_can_be_stopped | Switch | Indicates if this device can be stopped remotely. | Yes |
+| remote_control_can_be_paused | Switch | Indicates if this device can be paused remotely. | Yes |
+| remote_control_can_be_switched_on | Switch | Indicates if the device can be switched on remotely. | Yes |
+| remote_control_can_be_switched_off | Switch | Indicates if the device can be switched off remotely. | Yes |
+| remote_control_can_set_program_active | Switch | Indicates if the active program of the device can be set remotely. | Yes |
```

Comment:
Some of the Channels seems to describe properties rather than changing values. You could make those [properties](https://www.openhab.org/docs/developer/bindings/thing-xml.html#properties). Same might apply for the plate present Channels.

Reply:
We decided to use channels for the `remote_control_*` values because these are dynamically updated by the cloud based on the device state. Because of this we assumed them to be part of the "state" rather than a fixed "property". Do you think a property would be more suitable even if it is updated frequently? Our intend was to make scripting more comfortable and allow for generic rules. As far as I know properties cannot be accessed from within rules, is that still correct?

The `plate_is_present` channels are a different story, though. Making these properties is reasonable. What would be the preferred way in that case? Having one property `plate count` vs. having six boolean properties?

Reply:
Channel is just right for `remote_control_*`, then.

If the plate count is only an information, I'd go for `plate_count`.